CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Changed
```

Comment:
This should have one more `#`. While you're add it, can you make the `Added` in 0.6.1 have three? `###`